CONTRIBUTING.rst

```diff
@@ -387,37 +434,43 @@ checkout of MLflow:
       Generating ~/mlflow/mlflow/store/db_migrations/versions/b446d3984cfa_add_new_field_to_db.py
 
 
-These commands generate a new migration script (e.g. at
-``~/mlflow/mlflow/alembic/versions/12341123_add_new_field_to_db.py``) that you should then edit to add
-migration logic.
-
+These commands generate a new migration script (e.g., at ``~/mlflow/mlflow/alembic/versions/12341123_add_new_field_to_db.py``)
+that you should then edit to add migration logic.
 
-Launching the Development UI
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-We recommend `Running the Javascript Dev Server`_ - otherwise, the tracking frontend will request
-files in the ``mlflow/server/js/build`` directory, which is not checked into Git.
-Alternatively, you can generate the necessary files in ``mlflow/server/js/build`` as described in
-`Building a Distributable Artifact`_.
+Writing MLflow Examples
++++++++++++++++++++++++
+The ``mlflow/examples`` directory has a collection of quickstart tutorials and various simple examples that depict MLflow tracking,
+project, model flavors, model registry, and serving use cases. Written in `PEP8 Python coding style <https://www.python.org/dev/peps/pep-0008/>`_, these
+examples provide developers sample code, as a quick way to learn MLflow Python APIs.
 
+To ease your contributions’ review process, keep examples short yet demonstrative, add documentation, peruse through
+some ``mlflow/examples`` as templates for your contribution, and follow the recommended steps below.
 
-Running the Javascript Dev Server
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-`Install Node Modules`_, then run the following:
 
-In one shell:
+If you are contributing by adding new model flavor, follow these steps:
 
-.. code-block:: bash
+1. Follow instructions in `Python Model Flavors`_
+2. Create a corresponding directory in ``mlflow/examples/new-model-flavor``
+3. Implement your Python training ``new-model-flavor`` code in this directory
+4. Convert this directory’s content into an `MLflow Project <https://mlflow.org/docs/latest/projects.html>_` executable
+5. Add ``README.md``, ``MLproject``, and ``conda.yaml`` files and your code
+6. Read instructions in the ``mlflow/test/examples/README.md`` and add a ``pytest`` entry in the ``test/examples/test_examples.py``
+7. Add a short description in the ``mlflow/examples/README.md`` file
 
-   mlflow ui
+If you are contributing to the quickstart directory, follow these steps:
 
-In another shell:
+1. Create a descriptive Python program file with --help options
+2. Add a short description in the ``mlflow/examples/README.md`` file
+3. Read instructions in the ``mlflow/test/examples/README.md``, and add a ``pytest`` entry in the ``test/examples/test_examples.py``
```

Comment:
We don't actually want to encourage people to add new code to the quickstart (editing the existing quickstart is fine, but we should keep it brief) - maybe we should remove this section? Or just have a sentence stating that we welcome changes to the quickstart that make it clearer or simpler etc.